kafka-ui-api/src/main/java/com/provectus/kafka/ui/cluster/service/ConsumingService.java

```diff
@@ -55,6 +57,12 @@
 				.limitRequest(recordsLimit);
 	}
 
+	public Mono<Map<TopicPartition, Long>> loadOffsets(KafkaCluster cluster, List<TopicPartition> partitions) {
```

Comment:
we could do it simpler) just create consumer and get offsets.
Btw offset could be -1 for empty partitions

Reply:
Thanks, fixed